specs/http-api/README.md

```diff
@@ -109,41 +101,38 @@ If the serviceEndpoint is itself a DID, this DID should resolve to a document an
-| `code`             | N              | An application-specific error code, expressed as a string value.                                                                                                                                       |
-| `title`            | N              | A short, human-readable summary of the problem that SHOULD NOT change from occurrence to occurrence of the problem, except for purposes of localization.                                               |
-| `detail`           | Y              | A human-readable explanation specific to this occurrence of the problem. Like `title`, this field’s value can be localized.                                                                            |
-| `source`           | N              | An object containing references to the primary source of the error. It should include the `pointer`, `parameter`, or `header` members or be omitted.                                                   |
-| `source.pointer`   | N              | A JSON Pointer to the value in the request document that caused the error. This MUST point to a value in the request document that exists; if it doesn’t, the client SHOULD simply ignore the pointer. |
-| `source.parameter` | N              | A string indicating which URI query parameter caused the error.                                                                                                                                        |
-| `source.header`    | N              | A string indicating the name of a single request header which caused the error.                                                                                                                        |
-| `meta`             | N              | A meta object containing non-standard meta-information about the error.                                                                                                                                |
+## Error response structure
+| Field     | Type          | Required | Description                                                                            |
+| --------- | ------------- | -------- | -------------------------------------------------------------------------------------- |
+| `message` | String        | Y        | A human-readable explanation specific to this occurrence of the problem.               |
+| `id`      | String        | N        | Optional server-generated request-specific ID, useful for diagnosing unexpected errors |
+| `code`    | String        | Y        | An application-specific error code, expressed as a string value.                       |
+| `details` | ErrorDetail[] | N        | Optional array of `ErrorDetail` objects                                                |
+
+## Error `code`
+| Code | Description |
+| ---- | ----------- |
+| TODO |
```

Comment:
need input on the (exhaustive) list of codes to codify which code is used to describe which error scenario

Reply:
removing code for now